<ui version="4.0" >
 <class>FenClient</class>
 <widget class="QWidget" name="FenClient" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ZeroChat - Client</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>chat.png</normaloff>chat.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>IP du serveur :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="serveurIP" >
       <property name="maximumSize" >
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Port du serveur :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="serveurPort" >
       <property name="minimumSize" >
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum" >
        <number>1024</number>
       </property>
       <property name="maximum" >
        <number>65535</number>
       </property>
       <property name="value" >
        <number>50885</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boutonConnexion" >
       <property name="text" >
        <string>Connexion</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QLabel" name="label_5" >
         <property name="pixmap" >
          <pixmap>chat.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6" >
         <property name="font" >
          <font>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text" >
          <string>ZeroChat</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="listeMessages" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Pseudo :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pseudo" >
       <property name="maximumSize" >
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Message :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="message" />
     </item>
     <item>
      <widget class="QPushButton" name="boutonEnvoyer" >
       <property name="text" >
        <string>Envoyer</string>
       </property>
       <property name="icon" >
        <iconset>
         <normaloff>chat.png</normaloff>chat.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
